frontend/src/app.js

```diff
@@ -3,6 +3,7 @@
  */
 
 import algoliasearch from 'algoliasearch/lite';
+import 'alpinejs';
```

Comment:
I believe you need to install the package (i.e. add it to the `package.json`) to be able to import it like this.

Reply:
There were 2 options of installing alpinejs, either via CDN or NPM.
I followed the instructions below:

NPM installation
Include the following in your script using import alpinejs.
npm i alpinejs

Reply:
Right. The point is that `npm i alpinejs` should add it to `package.json`, but it was not the case. Now after I merged the main branch into yours, it appeared in `package.json` because it was added to the project in some other task in parallel to yours. So never mind it as of now.